### ISSUE TITLE
Make bash completion pip installable

### ISSUE THREAD

**Issue by alistair-broomhead** [russel/pip_bashcompletion#1]
I'm attempting to roll a package that will install pip autocompletion file on package install and gives a pip_update command to update the list - I'll reference this issue from the pull request

**Comment by papaloizouc** [comment]
yes sorry was going to add this as an issue (i realized after i used it) there's a grep on the index only which breaks the file autocomplete. i'm still wandering if we should merge with https://github.com/ekalinin/pip-bash-completion because it does some functionality we dont have, but because i didn't know how to fix this issue so i didn't make a pr there. 

The two repos can't be used together i think, also we need to change:
```
complete -o default -F _pip pip
```
to
```
complete -o default -F _pip pip
complete -o default -F _pip pip2
complete -o default -F _pip pip3
```

**Comment by alistair-broomhead** [comment]
ok - my current workings are at https://github.com/alistair-broomhead/pip_bashcompletion/compare/package?expand=1

**Comment by russel** [comment]
I think:
```
complete -o default -F _pip pip pip2 pip3
```
would be better. I have this is my feature branch and it works fine.

**Comment by papaloizouc** [comment]
:+1:  great looks good, since the other repo is not in pypi, when your pr is meged, will move the other repo here and let the author know (has no licence either :scream_cat:)

if you need help with the files autocomplete let us  know, will check what apt-get does.

**Comment by alistair-broomhead** [comment]
@russel this issue and its related pull request have been inactive without comment for some time, and are the only thing standing in the way of producing a package for pypi.

I'm happy to wait, but if you don't have time to maintain the project could you please add me as a contributor so I can push this through myself? I'd base the package off my own fork, but seeing as most of the hard work was done by yourself and @papaloizouc it would feel somewhat disingenuous for it not to be hosted from one of your Github accounts.

**Comment by papaloizouc** [comment]
Sorry guys promised to work on the asyncio staff on sunday but was too busy, will pick it at some point this week.

I don't mind for you getting the repository i was just surprised that this hasn't been done by anyone yet and its a nice feature. I guess @russel can add you as a collaborator if he is busy
